mock-abis/README.md

```
@@ -1,236 +1,202 @@
+    --add-opens java.base/java.lang.stream=ALL-UNNAMED \
+    --add-opens java.base/java.time=ALL-UNNAMED \
+    --add-opens java.base/java.time.LocalDate=ALL-UNNAMED \
+    --add-opens java.base/java.time.LocalDateTime=ALL-UNNAMED \
+    --add-opens java.base/java.io.Reader=ALL-UNNAMED \
+    --add-opens java.base/java.util.Optional=ALL-UNNAMED \
+    --add-opens java.base/java.time.LocalDateTime.date=ALL-UNNAMED \
+    -jar target/mock-abis-1.3.0-SNAPSHOT.jar
+    ```
+
+### 3. Flags
+| Flag | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)